examples/blog/package.json

```diff
@@ -6,19 +6,18 @@
     "start": "next start"
   },
   "dependencies": {
-    "gray-matter": "^4.0.2",
+    "gray-matter": "^4.0.3",
     "next": "latest",
-    "nextra": "^2.0.0-beta.5",
-    "nextra-theme-blog": "^0.1.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "nextra": "^1.1.0",
```

Comment:
Why were we downgrading here? This was bumped here: https://github.com/vercel/next.js/pull/32256


@shu do you have a preference?

Reply:
@balazsorban44 I reverted nextra back to the latest beta release.